api/v1beta1/kafkacluster_types.go

```diff
@@ -627,7 +627,7 @@ func (kSpec *KafkaClusterSpec) GetClusterImage() string {
 	if kSpec.ClusterImage != "" {
 		return kSpec.ClusterImage
 	}
-	return "ghcr.io/banzaicloud/kafka:2.13-2.8.1"
+	return "ghcr.io/banzaicloud/kafka:2.13-3.1.0"
```

Comment:
I am wondering if we should make this default container image a variable so it can be used in the common_test.go, instead of having this as the same string in multiple places

Reply:
We could, but does it matter really? We would be using it in about 2 places and changing the version is just a replace all basically here.

Reply:
"`but does it matter really?`"

Well, I think it's always better to have the same long strings refer to the common variable so we can avoid mistakes like possible typos (maybe this is just my naive opinion)

But yea, it has nothing to do with what we aim to do in the PR, just noticed it while reviewing the changes and thought we might take the chance to make our codebase a little more robust